tests/visual/taskboard-card.html

```diff
@@ -0,0 +1,217 @@
+<!doctype html>
+<html lang="en" class="k-typography">
+<head>
+    <title>TaskBoard</title>
+    <meta charset="utf-8" />
+    <link rel="stylesheet" data-role="kendo-theme" href="../../packages/default/dist/all.css" />
+    <link rel="stylesheet" href="assets/styles.css" />
+    <script src="assets/scripts.js"></script>
+
+    <style>
+        #test-area {}
+    </style>
+</head>
+<body>
+    <div id="test-area" class="grid k-grid-cols-1">
```

Comment:
This is not relevant for this specific test :) 

If we use Kendo utils for our tests, why do we continue using `grid` and not - `k-d-grid`? @joneff thought?

Reply:
Changed all tests to utilize the `.k-d-grid` class